src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -8,21 +8,16 @@ namespace APIViewWeb
 {
     public class PythonLanguageService : LanguageProcessor
     {
-        private static readonly string PythonHome = Environment.GetEnvironmentVariable("PYTHONHOME") ?? string.Empty;
+        private static readonly string PythonHome = Environment.GetEnvironmentVariable("APIVIEW_PYTHONHOME") ?? string.Empty;
```

Comment:
Let's get this value via IConfiguration so it can be set both from config or environment variable. 

See https://github.com/Azure/azure-sdk-tools/blob/999bab422926bd9be37d3eb304aa888360f5ec0c/src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs#L18 for an example.